src/Middleware/AuthorizationMiddleware.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Changing the formatting is going to cause CI to fail. Could you use `vendor/bin/phpcbf` to reformat the modified files to match the project's coding conventions?

Reply:
Done